### ISSUE TITLE
New Agent stuck on update firmware

### ISSUE THREAD

**Issue by kareltucek** [UltimateHackingKeyboard/agent#2030]
Agent keeps altering between "Update your firmware" and "loading keyboard configuration".

![2023-07-11-172559_1276x2128_scrot](https://github.com/UltimateHackingKeyboard/agent/assets/3447497/50b7c938-db5d-43ba-a9a8-e6b42662d720)

Then after powercycling the keyboard a new error popped up:

![2023-07-11-172841_1277x2124_scrot](https://github.com/UltimateHackingKeyboard/agent/assets/3447497/8d65e697-d130-4d8c-9f6b-feffffe6cb09)

But Agent still continues altering between update your firmware and loading keyboard configuration...

**Comment by kareltucek** [comment]
I solved it for myself somehow, so feel free to close... might have been some artifact from development...

**Comment by kareltucek** [comment]
ALso another cute bug: 

https://github.com/UltimateHackingKeyboard/agent/assets/3447497/033046e9-6485-4272-8486-b704dd211dbd

**Comment by kareltucek** [comment]
Actually, even after flashing new firmware multiple times from Agent and also manually from commandline, Agent still keeps on insisting that I need to update firmware

**Comment by kareltucek** [comment]
Also, keyboard is stuck on QWR keymap and Agent leaves me no way to reset the configuration...

**Comment by kareltucek** [comment]
Nope, not helping.

**Comment by mondalaci** [comment]
Try to close Agent, flash the right-side firmware from the command line, restart Agent, and reset your configuration if needed.

**Comment by kareltucek** [comment]
Well, I tried to download Agent 2.1.2 and unbrick the unit. The Agent did not let me through to the unbricking screen when shorting the pins as it should, even though UHK clearly rebooted multiple times.

After downgrading manually to 9.3.0. (via commandline), I managed to restore UserConfig to 5.1.0. using Agent 2.1.2, but now even Agent 2.1.2 insists on updating the firmware. Which I do, therefore falling from 9.3.0 (probably with custom hash) to 9.2.2. Even after this downgrade and restarting Agent multiple times, the Agent (2.1.2) still wants to update Firmware again and again.
